components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -10,6 +10,8 @@ import "mojo/public/mojom/base/values.mojom";
 import "url/mojom/origin.mojom";
 import "url/mojom/url.mojom";
 
+const int8 kAccountNameMaxCharacterLength = 30;
+
```

Comment:
Android changes will come in another PR?
cc: @simoarpe

Reply:
There's no need as Android handles accounts through WebUI now.